packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples/Datagrid.1.doc.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ExampleTemplate from 'terra-doc-template/lib/ExampleTemplate';
```

Comment:
was using the internal `ExampleTemplate` intentional? we could make use of the `terra-doc-template` and use `title` and `example` props to give context to examples

Reply:
The DocTemplate adds a bunch of borders around things that looks great on a doc page but kind of weird with my fullscreen-ish examples. If the ExampleTemplate is off limits, I can switch.